various-receipes/nestjs/ts-jest.config.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../jest.config');
+
+module.exports = {
+  ...config,
+  testMatch: [
+    '**/various-receipes/nestjs/app/test/basic-tests.test.ts',
```

Comment:
Neat - Maybe someone will learn from this that she has to configure her test patterns like this, one by one, prefer a more global pattern (**/*test.ts*...)